frontend/ui/widget/bookmapwidget.lua

```diff
@@ -522,13 +522,15 @@ local BookMapWidget = InputContainer:new{
     -- Extra symbols to show below pages
     extra_symbols_pages = nil,
 
-    _mirroredUI = BD.mirroredUILayout(),
-
     -- Make this local subwidget available for reuse by PageBrowser
     BookMapRow = BookMapRow,
 }
 
 function BookMapWidget:init()
+    if self.ui.view.inverse_reading_order then
+        BD.invert()
```

Comment:
I really thing BD.invert() should be plain (do what is requested without looking at a setting), and here:
`if self.ui.view.inverse_reading_order and G_reader_settings:isTrue("invert_ui_layout_mirroring")  then`
(with a better name for the 2nd setting).

Or it should be named `BD.invertIfRequested()` and just called here, and any checks for some combination of settings (so, possible others unrelated to inverse_reading_order) should all be made in this `BD.invertIfRequested()`.

The first solution feels more explicite: we would know here what is happening.
Or wrap all this in a `ReaderView:shouldInvertBiDiLayoutDirection()` and here:
`if self.ui.view:shouldInvertBiDiLayoutDirection() then`